### ISSUE TITLE
Plug in stops working if SSO is enabled on Tableau Server

### ISSUE THREAD

**Issue by wrsuarez** [schubergphilis/tableau-confluence-plugin#6]
If SSO is enabled on Tableau server this plug in gets the following error:

Error rendering macro 'tableau-plugin' : com.atlassian.renderer.v2.macro.MacroException: authentication error: there is an authentication problem while retrieving the report, please contact your confluence or tableau server administrator

It appears that there is no way to perform this type of auth once SSO is enabled and there's no provision for passing a password or calling an iDP to get a SAML token. Any chance this plug in will be updated?

**Comment by reevery** [comment]
If you have SSO, you should be able to embed the dashboard using the HTML plugin with the embed code. This plugin handles sign-in using Trusted Ticket, so will always reset any existing session. If there's a way to disable the authentication call, that will get it working for you, assuming you're logged in.

**Comment by roelgerrits** [comment]
Yes you can configure a key/value with keyname: disableclienttrustedauth and value: true.

Then when rendering the plugin it skips the trusted authentication.
